integration-test/src/main/resources/testsuites/e2e/aws-e2e-tests.yaml

```diff
@@ -12,6 +12,7 @@ tests:
       - name: com.sequenceiq.it.cloudbreak.testcase.e2e.environment.NewNetworkWithNoInternetEnvironmentTests
       - name: com.sequenceiq.it.cloudbreak.testcase.e2e.freeipa.FreeIpaTests
       - name: com.sequenceiq.it.cloudbreak.testcase.e2e.freeipa.FreeIpaRebuildTests
+      - name: com.sequenceiq.it.cloudbreak.testcase.e2e.freeipa.FreeIpaScalingTests
```

Comment:
why is this AWS only? we have to ensure the feature works on all provider

Reply:
I agree. This is because I could not find freeipa yaml files for azure andd I thought it was because to reduce test costs. Should then we add a freeipa test suite for azure as well?

Reply:
I think we should test everything on all provider as I see larger activity in freeipa related actions on customer side, like repair. I know it would increase the costs. Maybe combining the repair and scaling test would help. Like do the scaling, then as a last step do a repair on an instance

Reply:
Ok. Given that it would affect other freeipa tests as well, would it suffice to add the scaling tests only for freeipa to azure / gcp in this commit and in a separate one all the others?

Reply:
sure